sentinel/sentinel.go

```diff
@@ -63,7 +64,7 @@ func (p Proxy) handleRequestAndRedirect(w http.ResponseWriter, r *http.Request)
 		// add username/password to request
 		host = fmt.Sprintf("thorchain:password@%s:8332", host)
 	case "arkeo-mainnet-fullnode":
-		host = "arkeo:1317"
```

Comment:
curious on this change to arkeod for the host?

Reply:
I am using docker compose to test ,  https://github.com/arkeonetwork/arkeo/blob/master/docker-compose.yml#L3 , the name of arkeo is `arkeod` 

I am not sure how you guys test it.   does `arkeo:1317` work for you?